ntbs-service/package.json

```diff
@@ -12,6 +12,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "accessible-autocomplete": "^2.0.1",
```

Comment:
We've pinned other packages to a specific version, we should probably do the same here (remove the carat).

Reply:
good spot!